provider/cmd/pulumi-resource-synced-folder/s3-bucket-folder.ts

```diff
@@ -33,7 +34,7 @@ export class S3BucketFolder extends pulumi.ComponentResource {
         args.managedObjects = args.managedObjects ?? true;
         args.disableManagedObjectAliases = args.disableManagedObjectAliases ?? false;
```

Comment:
```suggestion
        args.disableManagedObjectAliases = args.disableManagedObjectAliases ?? false;
        args.includeHiddenFiles = args.includeHiddenFiles ?? false;
```

Reply:
Shall I still leave the default value in the function signature?